### ISSUE TITLE
Save sometimes needs to be clicked more than once

### ISSUE THREAD

**Issue by RussianRonin** [GetLuna/Luna#1617]
When I am editing settings, sometimes it does not save on first "save". I have to try 2 or 3 times for it to actually save.

**Comment by Studio384** [comment]
I'll be looking into this. Does this happen on specific pages?

**Comment by RussianRonin** [comment]
Mainly on the "Settings" page itself.

**Comment by Studio384** [comment]
Does it reload and just not save your settings or do you just need to doubleclick?

What OS/Browser combination are you using?

**Comment by RussianRonin** [comment]
It reloads and does not save settings.

Chrome 53 on Windows 10 Pro 64 bit.

**Comment by Studio384** [comment]
Does this always happen when trying to save the same setting (if yes, which?)?

**Comment by RussianRonin** [comment]
Yes. Description. I added an "!" and had to save 3 times (readding it everytime) for it to register and to remove it I had to save 2 times.

**Comment by Studio384** [comment]
I'm unable to reproduce this issue.

**Comment by Cotopaxi42** [comment]
I have the same issue, as described by RussianRonin.
